include/pros/colors.hpp

```diff
@@ -0,0 +1,168 @@
+/**
+ * \file pros/colors.hpp
+ *
+ * Contains enum class definitions of colors
+ *
+ * This file should not be modified by users, since it gets replaced whenever
+ * a kernel upgrade occurs.
+ *
+ * Copyright (c) 2017-2022 Purdue University ACM SIGBots.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License v. 2.0. If a copy of the MPL was not distributed with this
+ * file You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+#ifndef _PROS_COLORS_HPP_
+#define _PROS_COLORS_HPP_
+
+#define dark_grey dark_gray
```

Comment:
```suggestion
#define dark_grey dark_gray
```
We should make these enum values instead of macro definitions.